notebooks/speculative-sampling/speculative-sampling.ipynb

```diff
@@ -75,9 +75,17 @@
    "metadata": {
```

Comment:
<p>it is better to keep all dependencies install in one place at the notebook beginning, please move install penvino-genai from nightly instead of released dependencies (anyway openvino-genai from nightly required openvino and openvino_tokenizers from nightly too) </p>
<br/>

 _Reply via <a href='https://app.reviewnb.com/openvinotoolkit/openvino_notebooks/pull/2609/discussion/'>ReviewNB</a>_ <div id='ReviewNBCommentContext-DoNotDelete' style='display:none' data-state='OPEN' data-cellIndex='18'/>